model/src/main/proto/arguments.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+
+package model;
+
+option java_package = "org.oppia.android.app.model";
+option java_multiple_files = true;
+
+// Represents the arguments associated with ExitProfileDialogFragment.
+message ExitProfileDialogArguments {
+  // The last checked menu item needs to be restored or not.
+  bool restore_last_checked_menu_item = 3;
+
+  // The request for switching/exiting profile was made from Administrator Controls
+  // Fragment/Activity or not.
+  bool is_administrator_controls_selected = 2;
+
+  // The id of the menu item that was checked before the Exit Profile Dialog appears.
+  int32 last_checked_menu_item_id = 1;
```

Comment:
Is it possible to consolidate these into a single value the represents the place from which we opened the exit profile dialog?

Reply:
Done.